### PR TITLE
[Drumthumper] Code improvements

### PR DESCRIPTION
**Pull request by antonioolf**
Applying code improvements suggested by Android Studio

- Use of property access syntax instead of setter method;
- Make some methods private;
- Convert concatenation to template ;
- Removed unused variables and parameters;
- Updated the libraries `androidx.appcompat:appcompat` and `androidx.core:core-ktx` to the last versions.

Successfully tested on Android Studio 4.0.1

### CODE REVIEW

File: samples/drumthumper/src/main/java/com/plausibleaudio/drumthumper/TriggerPad.kt [hunk 1]
```diff
@@ -54,43 +53,39 @@ class TriggerPad: View {
...
 
     //
     // Layout Routines
     //
     private fun calcTextSizeInPixels() : Float {
-        val scaledSizeInPixels: Float = TypedValue.applyDimension(TypedValue.COMPLEX_UNIT_SP,
-                mTextSizeSp, resources.displayMetrics)
-
-        return scaledSizeInPixels;
+        return TypedValue.applyDimension(TypedValue.COMPLEX_UNIT_SP, mTextSizeSp, resources.displayMetrics)
```
Comment (philburk) [inline samples/drumthumper/src/main/java/com/plausibleaudio/drumthumper/TriggerPad.kt:81]: long line

File: samples/drumthumper/src/main/java/com/plausibleaudio/drumthumper/TriggerPad.kt [hunk 2]
```diff
@@ -100,24 +95,24 @@ class TriggerPad: View {
         // mTextSize = mDrawRect.bottom / 4.0f
     }
 
-    override fun onMeasure (widthMeasureSpec: Int, heightMeasureSpec: Int): Unit {
-        var widthMode = MeasureSpec.getMode(widthMeasureSpec);
-        var width = MeasureSpec.getSize(widthMeasureSpec);
+    override fun onMeasure (widthMeasureSpec: Int, heightMeasureSpec: Int) {
+        val width = MeasureSpec.getSize(widthMeasureSpec)
+
+       /* var padLeft = getPaddingLeft() */
```
Comment (philburk) [inline samples/drumthumper/src/main/java/com/plausibleaudio/drumthumper/TriggerPad.kt:101]: We can probably just remove these.

**Review by philburk — CHANGES_REQUESTED** [review]
Thanks for the cleanup. A few minor nits.

File: samples/drumthumper/src/main/java/com/plausibleaudio/drumthumper/DrumThumperActivity.kt [hunk 3]
```diff
@@ -142,18 +142,17 @@ class DrumThumperActivity : AppCompatActivity(),
         findViewById<LinearLayout>(R.id.rideMixControls).setVisibility(showFlag)
         findViewById<LinearLayout>(R.id.crashMixControls).setVisibility(showFlag)
 
-        findViewById<Button>(R.id.mixCtrlBtn).setText(
-                if (mMixControlsShowing) "Hide Mix Controls" else "Show Mix Controls")
+        findViewById<Button>(R.id.mixCtrlBtn).text = if (mMixControlsShowing) "Hide Mix Controls" else "Show Mix Controls"
```
[the remark this answers is not in the document]
Comment (antonioolf) [reply samples/drumthumper/src/main/java/com/plausibleaudio/drumthumper/DrumThumperActivity.kt]: done

File: samples/drumthumper/src/main/java/com/plausibleaudio/drumthumper/TriggerPad.kt (hunk 1 above)
Comment (antonioolf) [reply samples/drumthumper/src/main/java/com/plausibleaudio/drumthumper/TriggerPad.kt]: done

File: samples/drumthumper/src/main/java/com/plausibleaudio/drumthumper/TriggerPad.kt (hunk 2 above)
Comment (antonioolf) [reply samples/drumthumper/src/main/java/com/plausibleaudio/drumthumper/TriggerPad.kt]: done

**Comment by antonioolf** [comment]
@philburk  Thank you very much for your attention and code review.
I just submitted the corrections. Feel free to request me if there is anything else.

**Comment by philburk** [comment]
Looks good. I assume the vars were changed to val because val is "final".
When Paul approves we can merge this.

**Review by philburk — APPROVED** [review]
I am OK if Paul is OK. It's his code.

**Comment by android-usb** [comment]
I'm OK with it.

**Comment by antonioolf** [comment]
@philburk and @android-usb Thank you! :smile:

**Pull request merged**